### PR TITLE
Evelyn

### PR DESCRIPTION
**Pull request by Evelyn02204**
For Mr.Rymer to review.

### CODE REVIEW

**Comment by carlosrymer** [comment]
@Evelyn02204 do you want me to give general feedback or would you like me to focus on a specific area?

**Comment by Evelyn02204** [comment]
Both if you can. If not, then specific.

File: Contact_Us_Page.css [hunk 1]
```diff
@@ -1,6 +1,43 @@
...
+
+/** Flexbox **/
+.flex-boxes {
+    display: flex;
+    background: whitesmoke;
+    flex-direction: column;
+    width:450px;
+    border: 4px solid gainsboro;
+    border-radius: 6px;
+    padding:8px;
```
Comment (carlosrymer) [inline Contact_Us_Page.css:22]: You can set the left padding here:

```
padding: 8px 8px 8px 20px;
```

File: Contact_Us_Page.css [hunk 2]
```diff
@@ -1,6 +1,43 @@
...
+    padding-left:20px;
+    flex-flow: column nowrap;
```
Comment (carlosrymer) [inline Contact_Us_Page.css:24]: You don't need the `flex-direction` property if you will be using `flex-flow`. `flex-flow` includes `flex-direction`.

File: Contact_Us_Page.css [hunk 3]
```diff
@@ -1,6 +1,43 @@
...
+    justify-content: space-around;
+    align-items: flex-start;
+}
+.social-media-img-top {
+    width:36px;
+}
+.flex-social-m {
```
Comment (carlosrymer) [inline Contact_Us_Page.css:31]: Curious: why the `flex` in this class selector and the one below?

File: Contact_Us_Page.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <head>
-  <link rel="stylesheet" type="text/css" href="theme.css" />
+  <link rel="stylesheet" type="text/css" href="Contact_Us_Page_.css" />
```
Comment (carlosrymer) [inline Contact_Us_Page.html:3]: You have an extra `_` here. The file name is `Contact_Us_Page.css`.

File: Contact_Us_Page.html [hunk 5]
```diff
@@ -10,55 +10,70 @@
...
-                </div>
-              </div>
-        </div>
+
+    <h1> Contact Us</h1>
+    <h2> Get in touch with us!</h2>
+
+    <div class="flex-social-m flex-boxes">
+      <h3>Support Us</h3>
+        <img class="social-media-img-top" src="instagram-logo.png" alt="image of instagram icon">
```
Comment (carlosrymer) [inline Contact_Us_Page.html:19]: In place of images, consider using [font awesome](https://fontawesome.com/) icons.

File: Contact_Us_Page.html [hunk 6]
```diff
@@ -10,55 +10,70 @@
...
+        <p class="link-text">Bhorosha Instagram: <a href = "https://www.instagram.com/bhoroshabangladesh/">www.instagram.com/bhoroshabangladesh</a></p>
+        <img class="social-media-img-top" src="facebook-logo.png" alt="image of facebook icon">
+        <p class="link-text">Bhorosha Facebook: <a href = "https://www.facebook.com/bhoroshabangladesh">www.facebook.com/bhoroshabangladesh</a></p>
+        <img class="social-media-img-top" src="strikingly-logo.jpg" alt="image of strinkingly icon">
+        <p class="link-text">Bhorosha Strikingly: <a href = "https://bhoroshabangladesh.mystrikingly.com">https://bhoroshabangladesh.mystrikingly.com</a></p>
+    </div>
+        
+    <div class="flex-email flex-boxes">
+      <h3>Email Us</h3>
+        <img class="founder-img-top" src="..." alt="image of one of the founders">
```
Comment (carlosrymer) [inline Contact_Us_Page.html:29]: Use the name in the alt attribute.

File: Contact_Us_Page.html [hunk 7]
```diff
@@ -10,55 +10,70 @@
...
+        Email address: tasmeeatahana@gmail.com <br>
+        Phone number: # 
+        </p>
+        <img class="founder-img-top" src="..." alt="image of one of the founders">
+        <p class="email-text">Ahreem Shah: <br>
+        Email address: ahreemshah16@hotmail.com <br>
+        Phone number: #
+        </p> 
+        <div class="email_action ">
+        <form action="/action_page.php">
```
Comment (carlosrymer) [inline Contact_Us_Page.html:50]: Indent the child elements.

**Pull request closed without merging**